src/main/java/net/sf/jabref/bibtex/CamelCaser.java

```diff
@@ -1,101 +0,0 @@
-package net.sf.jabref.bibtex;
-
-import java.util.HashMap;
-import java.util.Map;
-
-class CamelCaser {
```

Comment:
I also found the class a good example for good code. However, we decided to remove unused code. :neckbeard: